sdk/schemaregistry/schema-registry-avro/sample.env

```diff
@@ -9,3 +9,12 @@ SCHEMA_REGISTRY_GROUP=<Group name for schemas in registry>
 AZURE_TENANT_ID=<AD tenant id or name>
 AZURE_CLIENT_ID=<ID of the user/service principal to authenticate as>
 AZURE_CLIENT_SECRET=<client secret used to authenticate to Azure AD>
+
+# Used in samples that use Event Hubs. Retrieve these values from an Event Hub in the Azure portal.
+EVENTHUB_CONNECTION_STRING=<Event Hub connection string>
```

Comment:
I will double check, but I believe the release pipeline will not let me release `@azure/schema-registry-avro` without releasing `@azure/event-hubs@5.7.0-beta.2` first since the semver for the EH dep is `^5.7.0-beta.2`.